client/src/components/ModalAdd.svelte

```diff
@@ -48,16 +66,32 @@
 				});
 
 				toast.success('Oprettelse vellykket!');
-				showAddModal.set(false);
 			} else {
-				console.error(`Fejl ved oprettelse: ${result.message || 'Oprettelse mislykkedes'}`);
+				toast.error(result.data || result.error); // NB! = error message: 'User was not created: user already exists'
```

Comment:
Ret til error i stedet for data: res.status(500).send({ data
evt. søg på andre status-koder